docs/components/ImageModal.tsx

```diff
@@ -11,7 +17,7 @@ const ClickableImage = styled.img`
     }
 `;
 
-export const ImageModal = ({ src, alt }) => {
+export const ImageModal: FC<ImageModalProps> = ({ src, alt }: ImageModalProps) => {
```

Comment:
From what I've read and seen from using the library, fC causes a lot of typing issues and it seems that the community has had some discussion around dropping it. This might not be the best place to have this discussion but it might be an idea to stop using FC generally and add our own children prop types.

Reply:
Agreed, I've read about the downsides as well. I was thinking about replacing them all here as well, but this would have made the PR less about getting it compliant with the linter and more about improving it as well. It's already huge as it is. 

There are some really good alternative interfaces you can use when you want to use children in the react api as well :)

Reply:
I've also read about the downsides but I'm not clear on what the best alternative is, could you share what is your approach?

Reply:
So the alternative is to type the children yourself explicitly, make it clear what the component expects, if you need to match the current behaviour of FC you can type it with React.ReactNode (if I remember correctly).

I think React does now provide some better built in types as well but I'm less familiar with them. I'm going to resolve this thread based on what we've discussed, I agree that this isn't the place to fix this but just to aware that we might cause additional ts errors for end users with it.

Reply:
You can check out the typings for react (yes they are not that great to read) for how it's done internally. From my experience and reading, I'd suggest just making the function return a `ReactElement | null`, which is in line with what the function from [`FC`/`FunctionalComponent`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L544) returns.